sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx

```diff
@@ -66,11 +97,7 @@ const NetworksSwitcher: FC = () => {
 	}: ReactSelectOptionProps) => (
 		<ExternalLink href={link} onClick={onClick}>
 			<LabelContainer noPadding={!!prefixIcon}>
-				{prefixIcon === 'Optimism' && (
-					<PrefixIcon>
-						<OptimismIcon width={20} height={14} />
-					</PrefixIcon>
-				)}
+				{!!prefixIcon && activeChain && networkIcon(activeChain.name)}
```

Comment:
Instead of duplicating the `<PrefixIcon>` component in the `networkIcon` function above, we can use it here once like:

```
{!!prefixIcon && activeChain && (
  <PrefixIcon>
    {networkIcon(activeChain.name)}
  </PrefixIcon>
)}
```

Reply:
Thanks. It's a better solution. Will make the change.